synfig-core/po/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+file(READ LINGUAS LANGUAGES)
+
+set(PO_DIR ${SYNFIG_BUILD_ROOT}/share/locale)
+string(REPLACE "\n" ";" LANGUAGES "${LANGUAGES}")
+list(APPEND LANGUAGES ${LANGUAGES})
+
+foreach(LANGUAGE IN ITEMS ${LANGUAGES})
+    list(APPEND LANGUAGE_LIST "${CMAKE_CURRENT_SOURCE_DIR}/${LANGUAGE}.po")
+endforeach()
+
+find_package(Gettext)
+if(GETTEXT_FOUND)
+    gettext_process_po_files(
+            ALL
+            PO_FILES ${LANGUAGE_LIST}
+    )
+
+    foreach(LANGUAGE ${LANGUAGES})
+        add_custom_command(
+                TARGET pofiles_1 POST_BUILD
+                COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/${LANGUAGE}.gmo
+                ${PO_DIR}/${LANGUAGE}/LC_MESSAGES/synfig.mo
+        )
+
+        install(
```

Comment:
`gettext_process_po_files` can add INSTALL target by self, so you don't need to do that.

Reply:
I was thinking about using that INSTALL, however, it will not install the files as the bash script does. It won't rename those files to `synfig.mo` and `synfigstudio.mo` but instead, it will use the base names, so something like this.
![image](https://user-images.githubusercontent.com/34449856/83753144-61cf6900-a687-11ea-9d86-e886cf49997f.png)
In fact, to properly use this, we also need to pass the `lang` parameter, but then, using that method, it will result in a type of code which I sent before, like the one that had 108 pofiles targets.